### PR TITLE
TACOSTRAT-917 - DeqmDataReason

### PR DESCRIPTION
**Pull request by awatson1978**

### CODE REVIEW

**Comment by p9g** [comment]
Are the .jekyll-cache files needed?

**Comment by p9g** [comment]
Related to https://jira.hl7.org/browse/FHIR-35807

File: input/examples/CodeSystem-DeqmOperationOutcomeCodeSystem.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "resourceType": "CodeSystem",
+  "status": "active",
+  "content": "complete",
+  "name": "DeqmOperationOutcomeCodeSystem",
+  "id": "DeqmOperationOutcomeCodeSystem",
+  "title": "DeqmOperationOutcomeCodeSystem",
+  "description": "Codesystem for OperationOutcomes used in DEQM settings.",
+  "version": "0.1.0",
+  "url": "https://gitlab.mitre.org/awatson/deqm-sandbox/CodeSystem/DeqmOperationOutcomeCodeSystem",
```
Comment (p9g) [inline input/examples/CodeSystem-DeqmOperationOutcomeCodeSystem.json:10]:
```suggestion
  "url": "http://hl7.org/fhir/us/davinci-deqm/CodeSystem/temporary-codes-guidance-response-reason-codes",
```

File: input/examples/ValueSet-DeqmOperationOutcomeValueSet.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "name": "DeqmOperationOutcomeValueSet",
+  "id": "DeqmOperationOutcomeValueSet",
+  "title": "DeqmOperationOutcomeValueSet",
+  "description": "ended valueset for OperationOutcomes used in DEQM settings.",
+  "version": "0.1.0",
+  "url": "https://gitlab.mitre.org/awatson/deqm-sandbox/ValueSet/DeqmOperationOutcomeValueSet",
+  "compose": {
+    "include": [
+      {
+        "system": "http://terminology.hl7.org/CodeSystem/operation-outcome"
```
Comment (p9g) [inline input/examples/ValueSet-DeqmOperationOutcomeValueSet.json:13]:
```suggestion
        "system": "http://hl7.org/fhir/us/davinci-deqm/CodeSystem/temporary-codes-guidance-response-reason-codes"
```

File: input/examples/ValueSet-DeqmOperationOutcomeValueSet.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      },
+      {
```
Comment (p9g) [inline input/examples/ValueSet-DeqmOperationOutcomeValueSet.json:15]: Delete.

**Comment by p9g** [comment]
Move codes system and value set into a new folder input/vocabulary
Rename the files to be consistent with the revised names

File: input/examples/CodeSystem-DeqmOperationOutcomeCodeSystem.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{
+  "resourceType": "CodeSystem",
+  "status": "active",
+  "content": "complete",
+  "name": "DeqmOperationOutcomeCodeSystem",
```
Comment (p9g) [inline input/examples/CodeSystem-DeqmOperationOutcomeCodeSystem.json:5]:
```suggestion
  "name": "DeqmGuidanceResponseReasonCodeSystem",
```

File: input/examples/ValueSet-DeqmOperationOutcomeValueSet.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
+{
+  "resourceType": "ValueSet",
+  "status": "active",
+  "name": "DeqmOperationOutcomeValueSet",
```
[the remark this answers is not in the document]
Comment (awatson1978) [reply input/examples/ValueSet-DeqmOperationOutcomeValueSet.json]: Landed on `DeqmDataReasonValueSet` per last week's conversation.

File: input/examples/CodeSystem-DeqmOperationOutcomeCodeSystem.json (hunk 4 above)
Comment (awatson1978) [reply input/examples/CodeSystem-DeqmOperationOutcomeCodeSystem.json]: Landed on `DeqmDataReasonCodeSystem` per last week's conversation.

File: input/examples/ValueSet-DeqmOperationOutcomeValueSet.json (hunk 3 above)
Comment (awatson1978) [reply input/examples/ValueSet-DeqmOperationOutcomeValueSet.json]: Done.

File: input/examples/CodeSystem-DeqmOperationOutcomeCodeSystem.json (hunk 1 above)
Comment (awatson1978) [reply input/examples/CodeSystem-DeqmOperationOutcomeCodeSystem.json]: Landed on the following, which should resolve once the PR is merged into `master`.
https://build.fhir.org/ig/HL7/davinci-deqm/DeqmDataReasonCodeSystem

**Comment by awatson1978** [comment]
I'd feel more comfortable moving the terminology files into a terminology folder during a refactor in a separate pull request.  A bit of organization cleanup would be good for the IG, but the page logic is a bit speghetti-ish, and I don't want to move too much stuff around in this PR.

**Comment by brynrhodes** [comment]
Agree with not refactoring as part of this PR, but the code system and value set being added here are resources, not examples. Also updated them to follow the convention for other resources in the IG, as well as fixed the URLs to be correct.

**Pull request merged**